server/core/src/https/mod.rs

```diff
@@ -324,39 +327,55 @@ pub async fn create_https_server(
         // the connect_info bit here lets us pick up the remote address of the client
         .into_make_service_with_connect_info::<ClientConnInfo>();
 
-    let addr = SocketAddr::from_str(&config.address).map_err(|err| {
-        error!(
-            "Failed to parse address ({:?}) from config: {:?}",
-            config.address, err
-        );
-    })?;
+    if config.address.starts_with("/") {
```

Comment:
I wonder if this function should take in a unix socket directly that way in tests we can pass in one half of an anonymous socket pair instead of relying on paths.

Reply:
Discussed offline: doesn't need it, since we handle this with config-loading in testing already :smile: